subprojects/packagefiles/sdl2/meson.build

```diff
@@ -840,6 +840,11 @@ elif platform_is_windows
         sys_joystick += 'rawinput'
     endif
 
+    # workaround for https://github.com/libsdl-org/SDL/issues/5580
+    if cdata.get('HAVE_XINPUT_H', 0) == 0
+        cdata.set('HAVE_WINDOWS_GAMING_INPUT_H', 0)
+    endif
+
```

Comment:
This is the best we can do to deal with an SDL bug.

Reply:
I tried moving this into th header, but that just exposed another SDL bug

Reply:
The bugs have been fixed with https://github.com/libsdl-org/SDL/pull/5592, now its only a matter of waiting for a release

Reply:
Great!

Any timeframe on the release? Given the current release is barely a week old, will we be waiting for a while?

Reply:
there is no publicly available timeframe, but I'd say about 2-4 months at most.